style.css

```diff
@@ -1,6 +1,6 @@
-.page_head {
-  color: white;
-  background-color: black;
-  padding: 10px;
-  text-align: center;
-}
\ No newline at end of file
+.page_head {
+  color: white;
+  background-color: black;
+  padding: 10px;
+  text-align: center;
+}
```

Comment:
Nice to see you fixed the linter error on line 6 now your work is fine. 👍 good job.
Happy coding!! 🎆